valuable/src/enumerable.rs

```diff
@@ -25,6 +25,9 @@ use core::fmt;
 /// [`visit_named_fields()`]: Visit::visit_named_fields
 /// [`visit_unnamed_fields()`]: Visit::visit_unnamed_fields
 ///
+#[cfg_attr(
```

Comment:
It is fine that documentation or tests require all features. See https://github.com/rust-lang/futures-rs/pull/2216 for more.

Reply:
The background of this patch was that the Debian build system does build crates with the different features both enabled and not enabled,  so in order to avoid errors there i wrote this patch: https://salsa.debian.org/rust-team/debcargo-conf/-/blob/master/src/valuable/debian/patches/fix-example-and-doc-compile-errors.patch

And I thought that I should also provide it upstream to be a good community member :)

Reply:
to follow up, does this mean that you are not interested in this PR? That isn't a big problem for me, we can just close it if it won't get merged.